README.adoc

```diff
@@ -312,39 +384,41 @@ In addition, a few endpoint tests have been included for you to test the basic f
 `inventory` and `system` services. If a test failure occurs, then you might have introduced a bug into
 the code.
 
-FaultToleranceTest.java
+FaultToleranceIT.java
 [source, java, linenums, role='code_column']
 ----
-include::finish/src/test/java/it/io/openliberty/guides/faulttolerance/FaultToleranceTest.java[tags=ft_testing;!javadoc]
+include::finish/src/test/java/it/io/openliberty/guides/faulttolerance/FaultToleranceIT.java[tags=ft_testing;!javadoc]
 ----
 // =================================================================================================
 // Running the tests
 // =================================================================================================
 
-[role='command']
-include::{common-includes}/mvnverify.adoc[]
+=== Running the tests
+
+Since you started Open Liberty in development mode at the start of the guide, press the `enter/return` key to run the tests. You will see the following output:
```

Comment:
Change
"You will see the following output:"
To
"You see the following output:"

Reply:
Updated